app/controllers/player_inputs_controller.rb

```diff
@@ -12,9 +12,12 @@ def create
     # line only gets executed after any player submits the first input
     @round.update(phase: 2) if @round.phase == 1
 
+    # Find the latest player that was created from the current user
+    player = Player.where(user_id: @current_user.id).order(id: :desc).limit(1)
```

Comment:
@spioupiou we should be able to get the player in the instance with:
`Player.find_by(instance_id: @instance_id, user_id: current_user.id)`
or
`Player.find_by(instance: @instance, user: current_user)`

Doing any of the above, we won't need to `order` `limit` and extract out of an array anymore.